nixos/modules/services/networking/ntopng.nix

```diff
@@ -64,6 +72,23 @@ in
         '';
       };
 
+      redis.address = mkOption {
+        type = types.str;
```

Comment:
Missing default value?

Reply:
This one shouldn't have a default, but we set it [here](https://github.com/NixOS/nixpkgs/pull/159538/commits/249c38532449ebcb8c5d3f814049f98cad3cd190#diff-89bb9a9988f12cbf17c0ea5dd3e856e5b347ff33913ec9820878e93bc9f5aa2cR124) if local Redis instance is created. Fixed it to not use `mkDefault`.